content/tutorial/02-advanced-orders/01-eth-flow/01-create-eth-flow/README.md

```diff
@@ -1,131 +1,204 @@
+  const buyToken = '0x177127622c4A00F3d409B75571e12cB3c8973d3c';
+  const sellAmount = '1000000000000000000';
+
+  const quoteRequest: OrderQuoteRequest = {
+    sellToken,
+    buyToken,
+    sellAmountBeforeFee: sellAmount,
+    kind: OrderQuoteSideKindSell.SELL,
+    receiver: ownerAddress,
+    +++from: ownerAddress,+++
+    appData: appDataContent,
+    appDataHash: appDataHex,
+    +++signingScheme: SigningScheme.EIP1271,+++
+    +++onchainOrder: true,+++
+  }
+
+  +++const { quote, id: quoteId } = await orderBookApi.getQuote(quoteRequest);+++
+}
 ```
 
-## Create the transaction to the Settlement contract
+In addition to the `OrderQuoteRequest` fields we've used in the previous tutorial, we've added:
+
+- `from`: the address of the order's owner (this is required for `EIP1271` orders)
+- `signingScheme`: as Eth-flow orders are signed using the `EIP1271` scheme, we need to specify such a scheme
+- `onchainOrder`: the order is going to be created on-chain, so we need to specify this
 
-In the previous step, we created an order in the order-book, but it's not valid yet.
-To make it valid, we need to create a transaction to the Settlement contract that proves the ownership of the order.
+### `EthFlowOrder.Data` struct
 
+Now that we have a quote, we can go about creating an `EthFlow` order. Unfortunately no `EthFlowOrder` type exists in the SDK, so we will need to create one ourselves. Simply we create a type that extends the `UnsignedOrder` type (removing the fields that are not required for [`EthFlow` orders](https://beta.docs.cow.fi/cow-protocol/reference/contracts/periphery/eth-flow#ethfloworderdata)) and add a `quoteId` field:
+
+```typescript
+/// file: run.ts
+// ...
+
++++type EthFlowOrder = Omit<UnsignedOrder, 'sellToken' | 'sellTokenBalance' | 'buyTokenBalance' | 'kind' | 'signingScheme'> & {+++
++++  quoteId: number;+++
++++}+++
+
+export async function run(provider: Web3Provider): Promise<unknown> {
+  // ...
+}
 ```
-// Create the pre-signature transaction
-const presignCallData = settlementContract.interface.encodeFunctionData(+++'setPreSignature'+++, [
-  orderId,
-  true,
-])
-const presignRawTx = {
-  to: settlementContract.address,
-  data: presignCallData,
-  value: '0',
+
+Now that we have the `EthFlowOrder` type, and we have a quote, we can create an order:
+
+```typescript
+/// file: run.ts
+import type { Web3Provider } from '@ethersproject/providers';
++++import { BigNumber, Contract } from 'ethers';+++
+// ...
+export async function run(provider: Web3Provider): Promise<unknown> {
+  // ...
+    const order: EthFlowOrder = {
+    ...quote,
+    +++buyAmount: BigNumber.from(quote.buyAmount).mul(9950).div(10000).toString(),+++
+    receiver: ownerAddress,
+    appData: appDataHex,
+    quoteId,
+  }
 }
+```
+
+> The `buyAmount` has a slippage of 0.5% applied to it. This is to ensure that it is filled if trading very low amounts of `xDAI` for this tutorial. The slippage amount is arbitrary and can be changed.
```

Comment:
As there is no explicit requirement within the services (safe for the potential case of the automated refunder), I'm not sure about making a statement in any way. Additionally, the 2% has caused issues so I think it would be reasonable to allow the user to determine this (considering that it doesn't impact the settlement of their order).